src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="search_input_big form"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'search_input_big' could be renamed to better reflect its purpose or content.